### ISSUE TITLE
FxA Settings buttons break with longer languages

### ISSUE THREAD

**Issue by juliemccracken** [mozilla/fxa-content-server#5507]
FxA Settings buttons break with longer languages

- Viewed on iPhone SE at least
- Button label copy should never wrap or break
- Button min-width should match the longest label in a given language

![button-unresponsive](https://user-images.githubusercontent.com/17153650/30650396-a191f4d8-9dd7-11e7-89e9-283521be456c.png)

**Comment by vladikoff** [comment]
@ryanfeeley what's the UXing strategy here to fix those buttons?

**Comment by vladikoff** [comment]
from mtg: try css grid here to fix this

**Comment by bwinton** [comment]
I've got something kinda working in [a branch](https://github.com/bwinton/fxa-content-server/tree/settings-grid) if anyone wants to check it out, but I'd like Ryan to take a look before I make a PR. 🙂

**Comment by ryanfeeley** [comment]
@vladikoff Have reviewed with Blake. Looks great but can you check in other browsers? iOS users can only view in WebKit for example.

**Comment by bwinton** [comment]
Soooo… I've tried hacking around the various settings templates, and while I think it's _possible_ to change the html and css and javascript and tests around to make it work on Chrome, I don't think it would be worth the week of effort it would take. 😢 (I also looked into changing the page to use tables, but that's not quite going to work either.)

As an alternate "fix", perhaps we could make the width of the buttons be localizable, so that German could make it longer? Or set `white-space: nowrap;` on `.settings-button` in the CSS, and let German be cut off instead of wrapping?
